packages/launch/src/focused-launch/summary/index.tsx

```diff
@@ -179,8 +207,73 @@ const Summary: React.FunctionComponent< Props > = ( { siteId } ) => {
 							<Icon icon={ check } /> Accept payments
 						</li>
 					</ul>
-				</div>
+				</>
+			}
+		/>
+	);
+};
+
+type SummaryProps = {
+	siteId: number;
+};
+
+const Summary: React.FunctionComponent< SummaryProps > = ( { siteId } ) => {
+	const { title, updateTitle, saveTitle } = useTitle( siteId );
+	const sitePrimaryDomain = useSelect( ( select ) =>
+		select( SITE_STORE ).getPrimarySiteDomain( siteId )
+	);
+	const siteSubdomain = useSelect( ( select ) => select( SITE_STORE ).getSiteSubdomain( siteId ) );
+	const hasPaidDomain = true; //sitePrimaryDomain && ! sitePrimaryDomain?.is_subdomain;
+
+	const domainSearch = useDomainSearch();
+
+	// Prepare Steps
+	const renderSiteNameStep = ( index: number ) => (
+		<SiteNameStep
+			stepIndex={ index }
+			key={ index }
+			value={ title }
+			onChange={ updateTitle }
+			onBlur={ saveTitle }
+		/>
+	);
+	const renderDomainStep = ( index: number ) => (
+		<DomainStep
+			stepIndex={ index }
+			key={ index }
+			existingSubdomain={ siteSubdomain?.domain }
+			currentDomain={ sitePrimaryDomain?.domain }
+			initialDomainSearch={ domainSearch }
+			hasPaidDomain={ hasPaidDomain }
+		/>
+	);
+	const renderPlanStep = ( index: number ) => <PlanStep stepIndex={ index } key={ index } />;
+
+	// Steps that are not interactive (e.g. user has already selected domain/plan)
+	const disabledSteps = [ hasPaidDomain ? renderDomainStep : null ].filter(
+		( step ) => step !== null
+	) as ( ( index: number ) => JSX.Element )[];
+
+	// Steps that require the user interaction
+	const activeSteps = [
+		renderSiteNameStep,
+		hasPaidDomain ? null : renderDomainStep,
+		renderPlanStep,
+	].filter( ( step ) => step !== null ) as ( ( index: number ) => JSX.Element )[];
```

Comment:
This is the only difference in behaviour introduced by this PR.

Previously, if the user has a paid domain, the domain step was still rendered together with the other summary steps (but without a "step index" in the title) — see #47110 

With this PR, if the user has a paid domain, the domain step is shown at the top (as a "disabled step"). The remaining "active" steps are shown below.